projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.ts

```diff
@@ -15,11 +15,17 @@ import { TraceDetailWidgetModel } from './trace-detail-widget.model';
     <div class="trace-detail-widget-renderer fill-container">
       <div class="content" *htLoadAsync="this.data$ as data">
         <ht-span-detail [spanData]="data" [showTitleHeader]="false">
+          <ht-summary-value
+            class="summary-value"
+            icon="${IconType.SpanId}"
+            label="Entry Span ID"
```

Comment:
What if there isn't an entry span? A regular trace doesn't really have this data - is it something we even want to show or just always show the (correct) trace id?